### PR TITLE
chore: verify that the clusterrole is not needed anymore

### PR DESCRIPTION
**Pull request by MatousJobanek**
to verify: https://github.com/codeready-toolchain/toolchain-common/pull/137

# !!DO NOT MERGE!!

### CODE REVIEW

File: make/test.mk [hunk 1]
```diff
@@ -105,8 +105,8 @@ endif
 
 .PHONY: setup-toolchainclusters
 setup-toolchainclusters:
-	curl -sSL https://raw.githubusercontent.com/codeready-toolchain/toolchain-common/master/scripts/add-cluster.sh | bash -s -- -t member -mn $(MEMBER_NS) -hn $(HOST_NS) -s
-	curl -sSL https://raw.githubusercontent.com/codeready-toolchain/toolchain-common/master/scripts/add-cluster.sh | bash -s -- -t host -mn $(MEMBER_NS) -hn $(HOST_NS) -s
+	curl -sSL https://raw.githubusercontent.com/MatousJobanek/toolchain-common/remove-clusterrole/scripts/add-cluster.sh | bash -s -- -t member -mn $(MEMBER_NS) -hn $(HOST_NS) -s
```
Comment (xcoulon) [inline make/test.mk:108]: so I assume this is a temporary change? :)

Comment (MatousJobanek) [reply make/test.mk:108]: yes, please, do not merge this PR :-)

**Comment by MatousJobanek** [comment]
Closing since it was already verified

**Pull request closed without merging**